tests/console/login.go

```diff
@@ -152,7 +154,7 @@ func LoginToFedora(vmi *v1.VirtualMachineInstance) error {
 			&expect.Case{
 				// Using only "login: " would match things like "Last failed login: Tue Jun  9 22:25:30 UTC 2020 on ttyS0"
 				// and in case the VM's did not get hostname form DHCP server try the default hostname
-				R:  regexp.MustCompile(fmt.Sprintf(`(localhost|fedora|%s) login: `, vmi.Name)),
+				R:  regexp.MustCompile(fmt.Sprintf(`(localhost|fedora|%s|%s) login: `, vmi.Name, hostName)),
```

Comment:
@EdDev Does this look good for you?

Reply:
@zhlhahaha , could you please explain why you had to add it?
Is this related to the change in the fedora image?

Reply:
The tests are configuring the hostname for the VM, which affects the login text, resulting in "inbound login:" with "inbound" being the new hostname. As you can see in the following 
code https://github.com/kubevirt/kubevirt/blob/e41a085f63bb4d22ffc2ab9900813d1a55c92f69/tests/network/services.go#L253-L255
The previous code didn't account for this scenario and only allowed "localhost" and "fedora."

Reply:
OK, I think I got it now.
The prompt was treated by the Alpine login and now that it got moved to Fedora, it needs to be supported there as well.

I think this is a valuable information and it will be good if you can add it to the commit message, reasoning for the need to add it (for future us).

Reply:
Ok, good suggestion, I will add it to the commit message.